etc/kayobe/ansible/dashboard.overview.j2

```diff
@@ -0,0 +1,314 @@
+{% set counter = { 'counter1': 0, 'counter2': 1 } %}
```

Comment:
Maybe put this in `etc/kayobe/ansible/templates/dashboard.overview.j2`? It's already a bit cluttered in here